src/basic/architecture.c

```diff
@@ -136,7 +136,7 @@ int uname_architecture(void) {
                 if (streq(arch_map[i].machine, u.machine))
                         return cached = arch_map[i].arch;
 
-        assert_not_reached("Couldn't identify architecture. You need to patch systemd.");
```

Comment:
Meh, what does "You need to patch systemd" really mean? You have to look at the function anyway. And how often do you add a new architecture… once every 3 years or so?

Reply:
I was just thinking that it could have been a hint for porter, as in "here's where you need to add your new arch-specific bits", but given it's only one of many many places, probably not worth it